libs/ngx-charts-on-fhir/src/lib/range-selector/range-selector.module.ts

```diff
@@ -6,9 +6,10 @@ import { MatInputModule } from '@angular/material/input';
 import { MatButtonToggleModule } from '@angular/material/button-toggle';
 import { MatNativeDateModule } from '@angular/material/core';
 import { FormsModule } from '@angular/forms';
+import { TimeFrameSelectorComponent } from '../timeframe-selector/timeframe-selector.component';
 
 @NgModule({
-  declarations: [RangeSelectorComponent],
+  declarations: [RangeSelectorComponent, TimeFrameSelectorComponent],
```

Comment:
Timeframe selector should be a separate module. If an app does not use summary component, it does not need timeframe component.
Also, module structure should match the directory structure or else things will get very confusing.